src/main/java/edu/cornell/kfs/coa/businessobject/CuAccountGlobal.java

```diff
@@ -2,164 +2,903 @@
 
 import java.sql.Date;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 import org.apache.commons.lang.StringUtils;
 import org.kuali.kfs.coa.businessobject.Account;
 import org.kuali.kfs.coa.businessobject.AccountGlobal;
 import org.kuali.kfs.coa.businessobject.AccountGlobalDetail;
+import org.kuali.kfs.coa.businessobject.AccountType;
+import org.kuali.kfs.coa.businessobject.BudgetRecordingLevel;
+import org.kuali.kfs.coa.businessobject.Chart;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryAccount;
+import org.kuali.kfs.coa.businessobject.IndirectCostRecoveryType;
+import org.kuali.kfs.coa.businessobject.RestrictedStatus;
+import org.kuali.kfs.coa.businessobject.SubFundGroup;
+import org.kuali.kfs.sys.KFSPropertyConstants;
 import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.rice.krad.bo.GlobalBusinessObjectDetailBase;
 import org.kuali.rice.krad.bo.PersistableBusinessObject;
 import org.kuali.rice.krad.service.BusinessObjectService;
+import org.kuali.rice.krad.util.ObjectUtils;
+import org.kuali.rice.location.framework.campus.CampusEbo;
 
-public class CuAccountGlobal extends AccountGlobal {
+import edu.cornell.kfs.coa.service.GlobalObjectWithIndirectCostRecoveryAccountsService;
+import edu.cornell.kfs.module.cg.businessobject.InvoiceFrequency;
+import edu.cornell.kfs.module.cg.businessobject.InvoiceType;
+
+public class CuAccountGlobal extends AccountGlobal implements GlobalObjectWithIndirectCostRecoveryAccounts{
     
     private static final long serialVersionUID = 1L;
 
-    protected String majorReportingCategoryCode;
-    protected MajorReportingCategory majorReportingCategory;
-    
-    @Override
-    public List<PersistableBusinessObject> generateGlobalChangesToPersist() {
+    protected DateTimeService dateTimeService;
+    protected BusinessObjectService businessObjectService;
+    protected transient GlobalObjectWithIndirectCostRecoveryAccountsService globalObjectWithIndirectCostRecoveryAccountsService;
```

Comment:
It must have been because the third service contained some non-serializable objects.

Since the getters for these services also appear to be auto-initializing the variables if needed, I would suggest declaring all 3 as transient so that we're not passing them around when serializing CuAccountGlobal.